plugins/modules/oneview_server_profile_facts.py

```diff
@@ -0,0 +1,338 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+###
+# Copyright (2016-2020) Hewlett Packard Enterprise Development LP
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# You may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+###
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['stableinterface'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = '''
+---
+module: oneview_server_profile_facts
+short_description: Retrieve facts about the OneView Server Profiles.
+description:
+    - Retrieve facts about the Server Profile from OneView.
+version_added: "2.5.0"
+requirements:
+    - "python >= 2.7.9"
```

Comment:
for collections we support python 3.x (eg: 3.4.2)

Reply:
done